### ISSUE TITLE
Zooming of topos not well centered

### ISSUE THREAD

**Issue by birgander2** [theCrag/website#2145]
The new image systems seems to be much better. Especially sharpness of zoomed topos has significantly increased. Thanks for that! One small glitch I found is that the zoomed version is not centered correctly, with the result that sometimes the left part of a topo disappears when zooming, while on the right side an empty region remains.

**Comment by brendanheywood** [comment]
This might sorta be a feature - does it zoom into the image and keep the point you clicked on in the same place? If you zoom out and click somewhere does the end position change? It could still be a bug and ultimately I'd like to redo this similar to the image zoom on flickr.

**Comment by birgander2** [comment]
No, it is always the same. The right edge is about 1cm left of the window edge and the left one outside. The problem is that the leftmost routes are not visible in the zoomed version.

**Comment by brendanheywood** [comment]
Yeah the maths might be a bit off at certain sizes, can you please dump a screen shot here + the url of the page and also the dimensions of your window

**Comment by birgander2** [comment]
Oh sorry, now it is as you have described!? Did you just have changed something? Now it seems to keep the center point. Perfect!

Anyway:
http://www.thecrag.com/climbing/germany/kochel/area/321355833

**Comment by brendanheywood** [comment]
Perhaps you've resized the window since?

**Comment by brendanheywood** [comment]
Ah yup - I can reproduce that, I think it happens with highly panoramic aspect ratio pictures, so a maths bug somewhere

**Comment by Mdemaillard** [comment]
see related #1668
